mods/ra2/rules/proxy-actors.yaml

```diff
@@ -5,3 +5,4 @@ barracks.infiltrated:
 warfactory.infiltrated:
 	AlwaysVisible:
 	ProvidesPrerequisite:
+
```

Comment:
Hint: if you remove it, the utility will keep adding it every time you run it. It's a known bug/annoyance.
I'd suggest keeping it to reduce future diff noise.